### PR TITLE
Dw biomarker refactor

### PR DESCRIPTION
**Pull request by DanielLyleWilliamWarren**
- Overhaul of Biomarker structure

### CODE REVIEW

File: src/app/pages/quickMaps/pages/biomarkers/biomarkerStatus/biomarkerStatus.component.ts [hunk 1]
```diff
@@ -389,12 +200,11 @@ export class BiomarkerStatusComponent implements AfterViewInit {
       case 'table':
         if (this.selectedCharacteristic) {
           console.log('dataSelected');
-          this.generateTable();
```
Comment (jon571) [inline src/app/pages/quickMaps/pages/biomarkers/biomarkerStatus/biomarkerStatus.component.ts:392]: This looks suspicious, in that something was done but it isn't now.

**Review by jon571 — COMMENTED** [review]
So you're splitting this up into separate components.  It's a good thig to do but difficult to properly review from a code perspective (I appreciate that it's normally me that sends you this kind of review :-)

Easier to test for changes then have a look through the code for anything suspect. 

When I used the interface to populate it all the map seemed to work fine, but when I reloaded the page it didn't.
https://preview534--micronutrientsupport-tool.netlify.app/quick-maps/biomarker?country-id=MWI&mnd-id=I&measure=biomarker&data-level=household&age-gender-group=all

There's also a js error (on dev too https://develop--micronutrientsupport-tool.netlify.app/quick-maps/biomarker?country-id=MWI&mnd-id=I&measure=biomarker&data-level=household&age-gender-group=all) which might mask any problems.

**Comment by DanielLyleWilliamWarren** [comment]
> So you're splitting this up into separate components. It's a good thig to do but difficult to properly review from a code perspective (I appreciate that it's normally me that sends you this kind of review :-)
> 
> Easier to test for changes then have a look through the code for anything suspect.
> 
> When I used the interface to populate it all the map seemed to work fine, but when I reloaded the page it didn't.
> https://preview534--micronutrientsupport-tool.netlify.app/quick-maps/biomarker?country-id=MWI&mnd-id=I&measure=biomarker&data-level=household&age-gender-group=all
> 
> There's also a js error (on dev too https://develop--micronutrientsupport-tool.netlify.app/quick-maps/biomarker?country-id=MWI&mnd-id=I&measure=biomarker&data-level=household&age-gender-group=all) which might mask any problems.

Hi @bgsandan . I've done a bit more work and resolved the issues that @jon571 Jon highlighted. I've added you as a reviewer as @jon571 is on AL

**Review by bgsandan — APPROVED** [review]
Looks good :+1:

**Pull request merged**